Use `cargo auditable --frozen` and disable `CARGO_HOME` (already set by abuild)
---
diff --git a/meson.build b/meson.build
index 4ccc054..87367e8 100644
--- a/meson.build
+++ b/meson.build
@@ -25,6 +25,7 @@ glib_compile_schemas = find_program('glib-compile-schemas', required: true)
 desktop_file_validate = find_program('desktop-file-validate', required: false)
 appstream_cli = find_program('appstreamcli', required: false)
 cargo = find_program('cargo', required: true)
+cargo_auditable = find_program('cargo-auditable', required: true)
 
 version = meson.project_version()
 
diff --git a/src/meson.build b/src/meson.build
index bb53eef..b13e59c 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -23,6 +23,7 @@ run_command(
 
 cargo_options = [ '--manifest-path', meson.project_source_root() / 'Cargo.toml' ]
 cargo_options += [ '--target-dir', meson.project_build_root() / 'target' ]
+cargo_options += [ '--frozen' ]
 
 if get_option('profile') == 'default'
   cargo_options += [ '--release' ]
@@ -33,8 +34,6 @@ else
   message('Building in debug mode')
 endif
 
-cargo_env = [ 'CARGO_HOME=' + meson.project_build_root() / 'cargo-home' ]
-
 cargo_build = custom_target(
   'cargo-build',
   build_by_default: true,
@@ -45,9 +44,7 @@ cargo_build = custom_target(
   install_dir: bindir,
   depends: resources,
   command: [
-    'env',
-    cargo_env,
-    cargo, 'build',
+    cargo, 'auditable', 'build',
     cargo_options,
     '&&',
     'cp', 'target' / rust_target / meson.project_name(), '@OUTPUT@',
